### PR TITLE
Feature/e UI 7605

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7605

### Change description ###
Case File View Move takes a long time -- handle error and move allowMoving to false

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts [hunk 1]
```diff
@@ -259,6 +259,7 @@ export class CaseFileViewFolderComponent implements OnInit, OnDestroy {
     });
 
     dialogRef.afterClosed().subscribe(newCatId => {
+      console.log(newCatId, node);
```
Comment (johnbenjamin-hmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts:262]: @MateiHMCTS - console.log statement

**Review by johnbenjamin-hmcts — DISMISSED** [review]
@MateiHMCTS All looks good apart from one minor change. Approved.

**Review by GaryScottHMCTS — APPROVED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts [hunk 2]
```diff
@@ -76,15 +77,35 @@ export class CaseFileViewFieldComponent implements OnInit, AfterViewInit, OnDest
     this.currentDocument = JSON.parse(mediaViewerInfo);
   }
 
-  public moveDocument(data:any) {
+  public moveDocument(data: { document: DocumentTreeNode, newCategory: string }) {
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts:80]: @MateiHMCTS - function is missing a return type

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts [hunk 3]
```diff
@@ -76,15 +77,35 @@ export class CaseFileViewFieldComponent implements OnInit, AfterViewInit, OnDest
...
+      )
+      .subscribe(res => {
+        if (res) {
+          this.resetErrorMessages();
+          this.reloadPage();
+        }
     });
   }
 
+  public reloadPage() {
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts:101]: @MateiHMCTS - function is missing a return type

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts [hunk 4]
```diff
@@ -76,15 +77,35 @@ export class CaseFileViewFieldComponent implements OnInit, AfterViewInit, OnDest
...
+    location.reload();
+  }
+
+  public resetErrorMessages() {
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts:105]: @MateiHMCTS - function is missing a return type

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - just a few function return types missing; otherwise OK

**Review by Daniel-Lam — DISMISSED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by Daniel-Lam — APPROVED** [review]
@MateiHMCTS - re-approved

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by Daniel-Lam — DISMISSED** [review]
@MateiHMCTS - re-approved again

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**